packages/react/src/primitives/shared/constants.ts

```diff
@@ -515,6 +515,47 @@ export const ComponentClassObject = {
   },
 } as const;
 
+export const SharedText = {
+  Collection: {
+    SearchFieldLabel: 'Search',
+  },
+  CountryCodeSelect: {
+    ariaLabel: 'Country code',
+  },
+  Fields: {
+    ariaLabel: {
+      clearField: 'Clear search',
+    },
+  },
+  PaginationItem: {
+    invisibleLabel: {
+      currentPage: 'Current Page:',
+    },
+    ariaLabel: {
+      goToNextPage: 'Go to next page',
+      goToPage: 'Go to page',
+      goToPrevPage: 'Go to previous page',
+    },
+  },
+  SearchField: {
+    ariaLabel: {
+      search: 'Search',
+    },
+  },
+  ShowPasswordButton: {
+    ariaLabel: {
+      showPassword: 'Show password',
+      hidePassword: 'Hide password',
+    },
+  },
+  StepperField: {
+    ariaLabel: {
+      IncreaseTo: 'Increase to',
+      DecreaseTo: 'Decrease to',
+    },
+  },
+};
+
```

Comment:
Exactly, I want to do something here but I dare not to because it could break things. Same reason above. I can update the newly added one from Pagination

Reply:
Discuss offline, we will mark `ShareText` as deprecated and remove it in the future and use `ComponentText` internally moving forward.